processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
Each dataSource should be associated with an interval. What do you think about adding a new dimension 'dataSourceAndInterval'?